content/en/tracing/api_catalog/_index.md

```diff
@@ -0,0 +1,137 @@
+
+In addition, you can quickly identify who is the owning team of each endpoint, who is the on-call engineer, and how to reach them (email, Slack, PagerDuty) to help resolve incidents involving a specific API endpoint faster.
+
+The ownership information -- team, on call information, communication details -- is derived from the underlying Service definition supplied to the Service Catalog.
+
+{{< img src="tracing/api_catalog/api-catalog-team-details.png" alt="The team details panel in the endpoint details page, showing the name and communication information for the team that owns the endpoint, as defined in Service Catalog" style="width:100%;" >}}
+
+## Group APIs to express a feature or business logic
+
+In addition to tagging an endpoint in its details page, you can **group endpoints by adding tags to multiple endpoints at once**. Select multiple endpoint check boxes, and click **Edit tags** to provide labels that describe business logic, importance, or other useful grouping information. Applying these labels can help you quickly view and access groups of endpoints defined by your own criteria, and create assets such as monitors and dashboards for endpoint groups with common ground and expectations.
+
+For example, if you want to create latency alerts for endpoints that are particularly sensitive to performance problems, tag those endpoints with a tag like `Latency sensitive`. 
+
+<!-- screen cap of making the "Latency sensitive" group tktk  -->
+
+Or tag endpoints that handle sensitive data with a tag like `PII` and alert when endpoints with that tag have a `401 Unauthorized` response. Other examples of useful tags:
+
+- `critical to feature X`
+- `newly added - V2`
+- `contains password`
+- `contains PII`
+- Business logic (for example, `payments` )
+- Specific customer group (`subscribers`)
+
+When you add a tag, it appears in the list of facets on the left of the catalog. Click a facet to filter the list and add the tag to the Search query field.
+
+
+## Setting up
+
+Automatically built on top of APM data, API Catalog can be turned on with no additional setup required. Only APM instrumented services and supported libraries are discoverable. 
+
+To set up your API Catalog list:
+- [Instrument the services][3] with APM. 
+- [Define service ownership information][2] using Service Catalog.
+
+### Key terminology
+
+API
+: A set of protocol and tools that allow two applications to communicate.
+
+API endpoint
+: The address of a resource (URL) of a server or a service, that implements the set of rules defined in the API, often through an HTTP, RESTful API interface. The API endpoint responsible for making the API call response.<br /><br/>
+API endpoints in the API Catalog are comprised by the HTTP Method (for example, `GET`), the URL's Path (the structure of the location of the resource, such as `/payment/{shop_id}/purchase`) and the Service this resource serves (for example, `Payments`)<br /><br/>
```

Comment:
```suggestion
API endpoints in the API Catalog are comprised by the HTTP Method (for example, `GET`), the URL's Path (the structure of the location of the resource, such as `/payment/{shop_id}/purchase`) and the Service this resource serves (for example, `Payments`)<br /><br/>
```
Is this saying that API Catalog only supports HTTP API monitoring or is this more of a meta disection of how we built the API Catalog using HTTP endpoints, and this is what you would see if you are looking at it? Either way needs to be slightly rephrased to clarify.

Reply:
Oooh, looking at the next line, it looks like this should all be combined and be something like 

```The API Catalog supports HTTP endpoints (for example, `GET`). When monitoring these endpoints, you'll seethe endpoint, the URL's Path (the structure of the location of the resource, such as `/payment/{shop_id}/purchase`) and the service this resource serves (for example, `Payments`)<br /><br/>```

Reply:
Sort of? I agree this could be written better, so I'll take another crack at it.